src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -1250,7 +1287,8 @@ private void flushNetwork(
         SSLEngine tlsEngine,
         int bytesProduced,
         MessageConsumer networkTarget,
-        long networkId)
+        long networkId,
+        Runnable applicationCleaner)
```

Comment:
Rename `applicationCleaner` to `networkReplyDoneHandler`.

Reply:
done